API/common/service_impl/knowledge.py

```diff
@@ -181,5 +183,6 @@ def pass_verification(question, result):
         content = response["choices"][0]["message"]["content"]
         content = content.replace("\"", "").replace("'", "").lower()
         if content not in ['true', 'false']:
-            raise GPTInvalidResultException(content)
+            logger.error(f"JSONDecodeError: {content}")
```

Comment:
I'm curious why it's Json decode error.

Reply:
Add a new exception: `GPTInvalidBoolException`